.travis.yml

```diff
@@ -78,9 +78,12 @@ script:
 
  - python3 -m venv pyenv && . pyenv/bin/activate
  - pip3 install --upgrade pip
- - pip3 install cython # required because git does not contain _NetworKit
- - NETWORKIT_PARALLEL_JOBS=2 pip3 install -e .
- - pip3 install ipython # required for tests
+ # Required because git does not contain _NetworKit.
+ - pip3 install cython
+ # -v ensures that pip reports the cmake output; prevents a Travis timeout.
+ - NETWORKIT_PARALLEL_JOBS=2 pip3 -v install -e .
```

Comment:
Unfortunately, the -v option creates a total mess in the travis log, because of the exhaustive networkit python dependencies.

As far as I understand the test environment we could just reduce the whole passage to
```
- pip3 install cython ipython
- NETWORKIT_PARALLEL_JOBS=2 NETWORKIT_OVERRIDE_CXX=CXX python3 setup.py test
```

`setup.py test` pulls the `dependencies`, calls `egg-info` and calls `build_ext`
I added `NETWORKIT_OVERRIDE_CXX=CXX` here to ensure that we build with the desired compiler and our own "compiler detection" does not kick in

I think @manpen introduced the feature to test the python module in travis and may be able to comment on this